### PR TITLE
Assignment-1

### PR DESCRIPTION
**Pull request by meta-pulkit-saxena**

### CODE REVIEW

File: Web/JS/Assignment-1/Js/Registration.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		alert("Password Length should be greater than 8");
+	}
+	// Matching The Password and Confirm Password
+	if(document.getElementById("password").value != document.getElementById("confirmPassword").value) 
+	{
+		count=1;
+		alert("Password & Confirm Password mismatch!!");
+	}
+	// Selecting the States
+	if(document.getElementById("state") == "Select State" || document.getElementById("city").value == "Select City" )
```
Comment (shikhar-metacube) [inline Web/JS/Assignment-1/Js/Registration.js]: check state with value

File: Web/JS/Assignment-1/Js/Registration.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+		alert("State or City are not Selected!!");
+	}
+	// Checking the Conditions if user created or not
+	if(count==0)
+	{
+		alert("USER CREATED SUCCESSFUL");
+	}
+	else 
+	{
+		alert(document.getElementById("textBox").value);
```
Comment (shikhar-metacube) [inline Web/JS/Assignment-1/Js/Registration.js]: Why this alert????

File: Web/JS/Assignment-1/Js/Registration.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+	
+}
+
+// function to fill The cities
+
+function createCity() 
+{
+	var cityObject = document.getElementById("city");
+	var stateObject = document.getElementById("state");
+	switch(stateObject.value)
```
Comment (shikhar-metacube) [inline Web/JS/Assignment-1/Js/Registration.js]: Try to create options by using state and city JS map to get city values for selected state.

File: Web/JS/Assignment-1/Js/Registration.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+// Validation for Checking all the conditions
+
+function validate()
+ {
+	var count=0;
+	// First name Validation
+	if(document.getElementById("firstName").value.trim() == "")
+	{
+		count=1;
```
Comment (shikhar-metacube) [inline Web/JS/Assignment-1/Js/Registration.js:27]: For validation error do not use alert as displaying so much alert is irritating for user. Display errors below fields.

File: Web/JS/Assignment-1/Js/Registration.js (hunk 2 above)
Comment (meta-pulkit-saxena) [inline Web/JS/Assignment-1/Js/Registration.js]: Done

Comment (meta-pulkit-saxena) [inline Web/JS/Assignment-1/Js/Registration.js]: It is removed.

File: Web/JS/Assignment-1/Js/Registration.js (hunk 1 above)
Comment (meta-pulkit-saxena) [inline Web/JS/Assignment-1/Js/Registration.js]: Done.

File: Web/JS/Assignment-1/Js/Registration.js (hunk 4 above)
Comment (meta-pulkit-saxena) [inline Web/JS/Assignment-1/Js/Registration.js:27]: Done.

File: Web/JS/Assignment-1/Js/Registration.js (hunk 3 above)
Comment (meta-pulkit-saxena) [inline Web/JS/Assignment-1/Js/Registration.js]: Done.

**Pull request merged**